pkg/gui/commits_panel.go

```diff
@@ -56,6 +62,13 @@ func (gui *Gui) refreshCommits(g *gocui.Gui) error {
 		if err != nil {
 			return err
 		}
+
+		for _, commit := range commits {
```

Comment:
this logic should live inside the CommitListBuilder code (meaning we should pass in the DiffEntries object when we call NewCommitListBuilder)

Reply:
I move it into `NewCmmitListBuilder` :)